packages/connectivity/src/scp-cf/connectivity-service-types.ts

```diff
@@ -32,6 +32,11 @@ export interface ProxyConfiguration {
    * HTTP headers to be added to a request if tunneled through the proxy.
    */
   headers?: ProxyConfigurationHeaders;
+
+  /**
+   * A JWT for proxy authorization.
```

Comment:
[nth] I would assume any authorization header could be accepted by the server. So perhaps adjust the description to something more general: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Proxy-Authorization

Reply:
[q] Also there is a `Proxy-Authorization` property in the `headers` which is added I [here](https://github.com/SAP/cloud-sdk-js/blob/84976bfdc835dd4b4aeadefbc1d3577a33cd43cc/packages/connectivity/src/scp-cf/authorization-header.ts#L144) I thought to the request. Could you double check if you really need this new property.

Reply:
I assume you do not use the destination for the mail, but still we could use the existing property.

Reply:
Yes, you are right when we are talking about a "header", which saved in line 34.
However, when using non-http (e.g., socket) protocols, the header is not applicable any more.
JWT is a common auth mechanism nowadays, so I added this key.